build/mvn

```diff
@@ -67,17 +67,27 @@ install_app() {
   fi
 }
 
-# Install maven under the build/ folder
+# Determine the Maven version from the root pom.xml file and
+# install maven under the build/ folder if needed.
 install_mvn() {
-  local MVN_VERSION="3.3.9"
-  local APACHE_MIRROR=${APACHE_MIRROR:-'https://www.apache.org/dyn/closer.lua?action=download&filename='}
+  local MVN_VERSION=`grep "<maven.version>" "${_DIR}/../pom.xml" | \
+                     head -1 | cut -f2 -d'>' | cut -f1 -d'<'`
```

Comment:
Since you use `awk` below, would this find the version correctly? `awk -F '[<>]' '{print $3}'` Looks like it, but I don't know if it's much simpler. You might also use "-n1" consistently like below in the `head` command.